src/components/TutorialMetadata.tsx

```diff
@@ -104,70 +34,126 @@ const TutorialMetadata: React.FC<IProps> = ({ tutorial }) => {
   const address = frontmatter.address
 
   return (
-    <Container>
-      <TagsContainer>
-        <PillContainer>
+    <Flex
+      flexDirection="column"
+      justifyContent="space-between"
+      borderBottomWidth={{ base: 0, lg: "1px" }}
+      borderBottomStyle={{ lg: "solid" }}
+      borderBottomColor={{ lg: "border" }}
+    >
+      <Flex justifyContent="space-between" alignItems="center" w="full" mb={8}>
+        <Flex flexWrap="wrap" w="full">
           <TutorialTags tags={frontmatter.tags} />
-        </PillContainer>
-        <SkillPill isSecondary={true} className="" color="">
+        </Flex>
+        <Flex
+          as={Pill}
+          alignSelf="flex-start"
+          mb={2}
+          whiteSpace="nowrap"
+          isSecondary={true}
+          className=""
+          color=""
+        >
           <Translation id={getSkillTranslationId(frontmatter.skill)} />
-        </SkillPill>
-      </TagsContainer>
-      <HorizontalContainer>
+        </Flex>
+      </Flex>
+      <Flex
+        mb={6}
+        flexWrap="wrap"
+        mt={-4}
+        fontSize="sm"
+        color="text300"
+        justifyContent="flex-start"
+      >
         {author && (
-          <DataContainer>
-            <Emoji size={1} mr={`0.5em`} text=":writing_hand:" />
+          <Box mr={4}>
+            <Emoji fontSize="sm" mr={2} text=":writing_hand:" />
             {author}
-          </DataContainer>
+          </Box>
         )}
         {hasSource && (
-          <DataContainer>
-            <Emoji size={1} mr={`0.5em`} text=":books:" />
+          <Box mr={4}>
+            <Emoji fontSize="sm" mr={2} text=":books:" />
             <Link to={frontmatter.sourceUrl}>{frontmatter.source}</Link>
-          </DataContainer>
+          </Box>
         )}
         {published && (
-          <DataContainer>
-            <Emoji size={1} mr={`0.5em`} text=":calendar:" />
+          <Box mr={4}>
+            <Emoji fontSize="sm" mr={2} text=":calendar:" />
             {getLocaleTimestamp(intl.locale as Lang, published)}
-          </DataContainer>
+          </Box>
         )}
-        <DataContainer>
-          <Emoji size={1} mr={`0.5em`} text=":stopwatch:" />
+        <Box mr={4}>
+          <Emoji fontSize="sm" mr={2} text=":stopwatch:" />
           {Math.round(tutorial.fields.readingTime.minutes)}{" "}
           <Translation id="comp-tutorial-metadata-minute-read" />
-        </DataContainer>
-      </HorizontalContainer>
-      <HorizontalContainer>
+        </Box>
+      </Flex>
+      <Flex
+        mb={6}
+        flexWrap="wrap"
+        mt={-4}
+        fontSize="sm"
+        color="text300"
+        justifyContent="flex-start"
+      >
```

Comment:
Same as suggestion for line 60. And no problem creating a child component to refactor this duplication!